craft/e300/src/System.scala

```diff
@@ -34,8 +34,13 @@ class E300ArtyDevKitSystem(implicit p: Parameters) extends RocketSubsystem
   val maskROMConfigs = p(MaskROMLocated(location))
   val maskRoms = maskROMConfigs.map { MaskROM.attach(_, this, CBUS) }
 
-  val boot = BundleBridgeSource(() => UInt(32.W))
-  tileResetVectorNexusNode := boot
+  val boot = if (maskRoms.isEmpty) {
```

Comment:
Since I'm a bit paranoid about this, just wanted to double check that this `val bool` is not being used by any other repo, since the type has changed into an `Option`?

Reply:
I just added it, so I am sure.